core/ir/aarch64/codec.c

```diff
@@ -5511,21 +5511,29 @@ static inline bool
 decode_opnd_svemem_gpr_simm4_vl_xreg(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
```

Comment:
So any traces (or other tool results) over the last few months with SVE would have the wrong addresses -- and we put out the 10.0 release in the interim right?  So this should have a mention in the Changelog as a visible problem with 10.0 that was fixed.

Reply:
Done.